src/translations/strings.yaml

```diff
@@ -576,13 +576,17 @@ onboarding:
   privacySlideBody: Use these features for more protection. They may cause issues on some sites, so you can turn them off anytime in settings.
   devicesSlideHeader: Install Mozilla VPN on up to 5 devices
   devicesSlideBody: Scan this QR code using a tablet or mobile device’s camera to install the VPN on another device.
+  devicesSlideBodyMobile: Scan this QR code using a tablet or mobile device’s camera to install the VPN on another device. To install on a desktop computer, visit the Download page on the Mozilla VPN website.
```

Comment:
> tablet or mobile device’s camera 

Non-native speaker: shouldn't both have the genitive?

Reply:
@flodolo Well, the genitives look ok to me, but I think this sentence could be a bit more direct.

What about "To install the VPN on another tablet or mobile device, scan the QR code."?

Reply:
Works fine for me 👍🏼

Reply:
@MattLichtenstein sorry for the delay - yes, let's update on desktop too. Thank you!